tests/hf/test_tapering.py

```diff
@@ -499,3 +455,74 @@ def test_transform_hamiltonian(symbols, geometry, generator, paulix_ops, paulix_
     for i, term in enumerate(sorted_terms):
         assert np.allclose(term[0], ham_ref.terms[0][i])
         assert term[1].compare(ham_ref.terms[1][i])
+
+
+@pytest.mark.parametrize(
```

Comment:
You can add more tests if they are not expensive, similar to what you have [here](https://github.com/PennyLaneAI/pennylane/pull/2042).

Reply:
Added one test for HeH+. A bigger molecule like LiH would be expensive so not including it here.